.github/workflows/demo_apps_build.yaml

```diff
@@ -0,0 +1,107 @@
+            ZEPHYR_TOOLCHAIN_VARIANT: zephyr
+            ZEPHYR_SDK_INSTALL_DIR: /opt/telink/zephyr-sdk-0.15.2
+
+        runs-on: ubuntu-latest
+        if: github.actor != 'restyled-io[bot]'
+
+        steps:
+            - name: Install dependencies
+              run: |
+                    sudo apt-get install -fy --no-install-recommends ninja-build cmake make gcc gcc-multilib g++-multilib libsdl2-dev
+
+            - name: Setup toolchain
+              run: |
+                    mkdir /opt/telink \
+                    && cd /opt/telink \
+                    && wget https://github.com/zephyrproject-rtos/sdk-ng/releases/download/v0.15.2/zephyr-sdk-0.15.2_linux-x86_64.tar.gz \
+                    && wget -O - https://github.com/zephyrproject-rtos/sdk-ng/releases/download/v0.15.2/sha256.sum | shasum --check --ignore-missing \
+                    && tar xvf zephyr-sdk-0.15.2_linux-x86_64.tar.gz \
+                    && rm -rf zephyr-sdk-0.15.2_linux-x86_64.tar.gz \
+                    && cd zephyr-sdk-0.15.2 \
+                    && ./setup.sh -t riscv64-zephyr-elf -h -c
+
+            - name: Setup Zephyr sources
+              run: |
+                    echo "Creating 'zephyrproject' directory" \
+                        && mkdir /opt/telink/zephyrproject \
+                        && cd /opt/telink/zephyrproject \
+                    && echo "West setup" \
+                        && python -m venv .venv \
+                        && source .venv/bin/activate \
+                        && python -m pip install -U pip \
+                        && python -m pip install -U --no-cache-dir west==0.12.0 imgtool==1.7.0 \
+                    && echo "Initializing west ..." \
+                        && west init . \
+                    && echo "Cloning telink-semi/zephyr repository ..." \
+                        && cd zephyr \
+                        && git remote add telink "https://github.com/telink-semi/zephyr.git" \
+                        && git fetch telink \
+                        && git checkout telink/develop \
+                        && git checkout -b develop \
+                        && cd .. \
+                    && echo "continue west setup" \
+                        && west update \
+                        && west zephyr-export \
+                        && pip install -r zephyr/scripts/requirements.txt
+
+            - name: Build Telink samples/basic/blinky app
+              run: |
+                    cd /opt/telink/zephyrproject/zephyr/samples/basic/blinky \
+                    && source /opt/telink/zephyrproject/.venv/bin/activate \
+                    && west build -p -b tlsr9518adk80d
+
+            - name: Build Telink samples/bluetooth/peripheral_ht app
+              run: |
+                    cd /opt/telink/zephyrproject/zephyr/samples/bluetooth/peripheral_ht \
+                    && source /opt/telink/zephyrproject/.venv/bin/activate \
+                    && west blobs fetch hal_telink \
```

Comment:
Remove here "west blobs fetch hal_telink" and use above in Zephyr setup job

Reply:
Then it surprisingly doesn't work and I don't know why ...